Description: Use default settings for MAAS PSERV
 Use default settings for MAAS pserv. These default settings
 include port, logfile, oops directory, cobbler url and username
Author: Andres Rodriguez <andreserl@ubuntu.com>

--- a/etc/maas/pserv.yaml
+++ b/etc/maas/pserv.yaml
@@ -6,7 +6,7 @@
 ## running server.
 #
 # logfile: "pserv.log"
-logfile: "/dev/null"
+logfile: "/var/log/maas/pserv.log"
 
 ## OOPS configuration (optional).
 #
@@ -15,7 +15,7 @@
   # or directories other than what the oops machinery creates there.
   #
   # directory:
-  directory: "logs/oops"
+  directory: "/var/log/maas/oops"
   # reporter:
   reporter: "maas-pserv"
 
@@ -38,8 +38,6 @@
   # resource_root: /var/lib/maas/boot-resources/current/
 
   # port: 69
-  port: 5244
   ## The URL to be contacted to generate PXE configurations.
   # generator: http://localhost/MAAS/api/1.0/pxeconfig/
-  generator: http://localhost:5243/api/1.0/pxeconfig/
 
